cmd/orch-tester/orch_tester.go

```diff
@@ -549,3 +599,209 @@ func (s *streamerClient) getOrchestrators() ([]*orch, error) {
 
 	return data.Data.Transcoders, nil
 }
+
+// Get a random source/rendition sample segment and write it to external storage
+func (s *streamerClient) randomSample(ctx context.Context, mid string, orch string) error {
+	randSeqNo := rand.Int() % numSegments
+	randProfile := s.profiles[rand.Int()%len(s.profiles)]
```

Comment:
The profiles are defined first as a flag `presets` which is passed into the HTTP request for the stream-tester which I believe uses those presets for the transcoding. But they are not used on the broadcaster because the `-authWebhookUrl` is not defined on the broadcaster, right ? So yes in that case they would need to be the same.

Reply:
Got it. Yeah the only way to set the presets on the broadcaster is to either directly set them (i.e. via CLI, startup flags, etc.) or via a webhook. I believe the presets passed to stream-tester can be used for .com API stream configurations, but in this case since orch-tester is using the broadcaster directly I don't think the presets passed to stream-tester are actually used. So, the primary purpose of passing presets to orch-tester is for selecting the rendition to sample from an orchestrator.